Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -309,10 +309,18 @@ static void HandleQuery()
 {
   DEBUG_LOG_FMT(GDB_STUB, "gdb: query '{}'", CommandBufferAsString() + 1);
```

Comment:
Just as an opinion thing, I feel it would be clearer to not get rid of the first character (both here and in the `strcmp` calls) to make it more obvious what the command names are for searching in the documentation.